packages/mgt-components/src/components/mgt-person-card/mgt-person-card.scss

```diff
@@ -52,10 +53,24 @@ $person-card-details-item-font-size: var(--person-card-details-item-font-size, 1
     }
   }
 
+  .close-card-container {
```

Comment:
@beth-panx 

Just getting to these now... sorry for the delay. 😟

Thinking about this... If you use the close button with the standalone person-card, I think it would suggest to close the entire card? How would it be re-rendered? 

Or should we have it "reset" the view so that if its expanded, and the user uses the close button, it'll render the non-expanded view?

Maybe @yyiinnggzz has some ideas. Tagging for visibility. 

Standalone vs card:
https://user-images.githubusercontent.com/11411686/131753411-4bd07b77-d51c-4964-8790-f36cd1ee0f1f.mp4

Reply:
Resolving this, as we decided not to include for standalone